azure-cognitiveservices-vision-computervision/README.md

```diff
@@ -0,0 +1,287 @@
+# Azure Cognitive Services Computer Vision SDK for Python
+
+The cloud-based Computer Vision service provides developers with access to advanced algorithms for processing images and returning information. This service works with popular image formats, such as JPEG and PNG. 
+
+To analyze an image, you can either upload an image or specify an image URL. Computer Vision algorithms analyze the content of an image in different ways, depending on the visual features you're interested in. For example, Computer Vision can determine if an image contains adult or racy content, or find all the faces in an image.
+
+You can use Computer Vision in your application to:
+
+- Analyze images for insight
+- Extract text from images
+- Generate thumbnails
+
+Looking for source code or API reference?
+
+* [SDK source code][source_code]
+* [SDK reference documentation][ref_computervision_sdk]
+
+## Prerequisites
+
+* Azure subscription - [Create a free account][azure_sub]
+* Azure [Computer Vision resource][computer_vision_resource]
+* [Python 3.6+][python]
+
+If you need a Computer Vision API account, you can create one with this [Azure CLI][azure_cli] command:
+
+```Bash
+RES_REGION=westeurope 
+RES_GROUP=<resourcegroup-name>
+ACCT_NAME=<computervision-account-name>
+
+az cognitiveservices account create \
+    --resource-group $RES_GROUP \
+    --name $ACCT_NAME \
+    --location $RES_REGION \
+    --kind ComputerVision \
+    --sku S1 \
+    --yes
+```
+
+## Installation
+
+Install the Azure Cognitive Services Computer Vision SDK with [pip][pip], optionally within a [virtual environment][venv].
+
+### Configure a virtual environment (optional)
+
+Although not required, you can keep your base system and Azure SDK environments isolated from one another if you use a virtual environment. Execute the following commands to configure and then enter a virtual environment with [venv][venv], such as `cogsrv-vision-env`:
+
+```Bash
+python3 -m venv cogsrv-vision-env
+source cogsrv-vision-env/bin/activate
+```
+
+### Install the SDK
+
+Install the Azure Cognitive Services Computer Vision SDK for Python with [pip][pip]:
+
+```Bash
+pip install azure-cognitiveservices-vision-computervision
+```
+
+## Authentication
+
+Once you create your Computer Vision resource, you need its **region**, and one of its **account keys** to instantiate the client object.
+
+Use these values when you create the instance of the [ComputerVisionAPI][ref_computervisionclient]. 
```

Comment:
Since that class name is not very representative of what it is (a client object), adding "client object" at the end might help it pop a bit that that's what it is.

Use these values when you create the instance of the [ComputerVisionAPI][ref_computervisionclient] **client object**.

Reply:
I was wondering about this but was trying to use Cosmos DB as a standard. I'll fix this.